tests/errmsgs/tconfig_error.nim

```diff
@@ -0,0 +1,10 @@
+discard """
+  description: '''
+  Test that a config file with invalid option will yield an error with
+  non-zero exit code.
+
+  Ref https://github.com/nim-works/nimskull/issues/1112
```

Comment:
```suggestion
    Test that a config file with invalid option will yield an error with
    non-zero exit code.

    References https://github.com/nim-works/nimskull/issues/1112
```
Indenting the body makes the relationship easier to parse. I'd also suggest not abbreviating "References" in test specifications (or at least using "Refs").